Feature/define challenge

Webdev-Pibs opened the pull request.

Webdev-Pibs committed:
detailsPage => filters + sort done

Webdev-Pibs committed:
all comments

Webdev-Pibs committed:
Merge branch 'dev' into feature/define-challenge

Webdev-Pibs committed:
fix router

Webdev-Pibs committed:
refix router

BeatlesBOB commented on src/services/date.ts:
```diff
@@ -0,0 +1,12 @@
+        ss: date.getSeconds().toString().padStart(2, '0'),
+    };
+
+    return format.replace(/(dd|mm|yyyy|hh|nn|ss)/g, (match) => options[match]);
```
Utilise https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Plutot

Webdev-Pibs replied:
ptn bien vu merci je change ça

BeatlesBOB replied:
Tu a un composant Date aussi qui formate le temps en fonction de la langue du mec

BeatlesBOB commented on src/components/02 - Molecules/Comment/Comment.tsx:
```diff
@@ -0,0 +1,113 @@
+import ProfileButton from '@/components/01 - Atoms/ProfileButton/ProfileButton';
+import Button from '@/components/01 - Atoms/Button/Button';
+import Svg from '@/components/01 - Atoms/Svg/Svg';
+import Parser from 'html-react-parser';
```
Désinstaller le module et utiliser DomPurify

Webdev-Pibs committed:
euhhh un peu plus de pages détails mais voilà quoi

- uninstall react-parser car on a dompurify
- j'ai pas retouché aux dates par flemme
- début de fomurlaire de solution
- début d'affichage de description + images

Webdev-Pibs committed:
AHAHAHA CA FONCTIONNE

création des solutions

Webdev-Pibs committed:
pages solutions d'un challenge

Webdev-Pibs committed:
Merge branch 'dev' into feature/define-challenge
